test/tests/sawyercoding_test.cpp

```diff
@@ -0,0 +1,43 @@
+extern "C" {
+	#include "util/sawyercoding.h"
+}
+
+#include <gtest/gtest.h>
+
+// TODO: Seed with some proper randomness
+char randomdata[] = { 0x01, 0x02, 0x03, 0x04 };
```

Comment:
Why does the file use `char`, `uint8`, and `uint8_t`? Shouldn't one of those be preffered?